tests/densities/test_met_types.py

```diff
@@ -0,0 +1,109 @@
+#Tests
+
+import os
+import time
+import nrrd
+import numpy as np
+import pandas as pd
+from unittest.mock import patch, MagicMock
+
+## Unit Tests:
+def test_process_t_type():
+    # Synthetic test data
+    t_type = "test_t_type"
+    t_type_data = np.ones((10, 10, 10))  # Small synthetic voxel array
+    p_map_subset = pd.Series({"me_type_1": 0.5, "me_type_2": 0.5})
+    header_sizes = (10, 10, 10)
+
+    # Mock load_t_type_data to return synthetic data
+    with patch("your_module.load_t_type_data", return_value=t_type_data):
```

Comment:
Is this `your_module` a leftover placeholder? tests are failing with  
`ModuleNotFoundError: No module named 'your_module'`